pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -291,10 +291,12 @@ spec:
     kind: Service
     metadata:
       annotations:
+        openebs.io/pvc-namespace: {{ .Volume.runNamespace }}
```

Comment:
Can we rename this to `openebs.io/persistent-volume-claim-ref: |`

Reply:
cc @vishnuitta

Reply:
we can do this

Reply:
sorry.. trying to understand the annotation key..
If the value is 'namespace' name, current key `pvc-namespace` seems to be fine.